examples/options/src/options/common_options.cpp

```diff
@@ -26,6 +26,13 @@ traccc::common_options::common_options(po::options_description& desc) {
     desc.add_options()("check_performance",
                        po::value<bool>()->default_value(false),
                        "generate performance result");
+    desc.add_options()("detector_file", po::value<std::string>()->required(),
+                       "specify detector file");
+    desc.add_options()("material_file", po::value<std::string>(),
```

Comment:
I added `po::value<std::string>()` in material file option - otherwise it doesn't take any arguments